apps/dolly-frontend/src/main/js/src/pages/adminPages/Orgtilgang/RedigerOrganisasjon.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button } from '@navikt/ds-react'
+import OrganisasjonTilgangService from '@/service/services/organisasjonTilgang/OrganisasjonTilgangService'
+import { PencilIcon } from '@navikt/aksel-icons'
+import useBoolean from '@/utils/hooks/useBoolean'
+import React from 'react'
+import Icon from '@/components/ui/icon/Icon'
+import NavButton from '@/components/ui/button/NavButton/NavButton'
+import { formatDate } from '@/utils/DataFormatter'
+import { FormSelect } from '@/components/ui/form/inputs/select/Select'
+import { FormProvider, useForm } from 'react-hook-form'
+import { FormDatepicker } from '@/components/ui/form/inputs/datepicker/Datepicker'
+import './RedigerModal.less'
+import { DollyModal } from '@/components/ui/modal/DollyModal'
+
+type RedigerTypes = {
+	orgNr: string
+	gyldigTil: string
+	miljoe: string
+	mutate: Function
+}
+
+const miljoeOptions = [
+	{ label: 'Q1', value: 'q1' },
+	{ label: 'Q2', value: 'q2' },
+]
+
+export const RedigerOrganisasjon = ({ orgNr, gyldigTil, miljoe, mutate }: RedigerTypes) => {
+	const [modalIsOpen, openModal, closeModal] = useBoolean(false)
+	const formMethods = useForm({
+		mode: 'onChange',
+		defaultValues: {
+			miljoe: miljoe?.includes(',') ? miljoe.split(',') : [miljoe],
+			organisasjonsnummer: orgNr,
+			gyldigTil,
+		},
+	})
+
+	const values = formMethods.watch()
+
+	const updateOrganisasjon = () => {
+		OrganisasjonTilgangService.updateOrganisasjon(values).then(() => {
+			mutate()
+		})
+	}
+
+	return (
+		<FormProvider {...formMethods}>
+			<Button
+				onClick={openModal}
+				variant={'tertiary'}
+				icon={<PencilIcon />}
+				size={'small'}
+				style={{ marginLeft: '10px' }}
+			/>
+			<DollyModal isOpen={modalIsOpen} closeModal={closeModal} width={'50%'} overflow={'visible'}>
```

Comment:
Synes kanskje det blir litt feil at denne modalen er utformet som en warning som må bekreftes istedenfor en ren modal for redigering? 🤔 Passer bra når vi sletter ting, men her blir det litt dramatisk 😱 Ville nok heller utformet den mer lik "Opprett mal fra bestilling" for eksempel. Altså kun med tittel, felter og avbryt/lagre-knapper nederst 😎